pkg/webhook/validate.go

```diff
@@ -165,6 +165,13 @@ func staticValidateSriovNetworkNodePolicy(cr *sriovnetworkv1.SriovNetworkNodePol
 	if cr.Spec.VdpaType == constants.VdpaTypeVirtio && cr.Spec.EswitchMode != sriovnetworkv1.ESwithModeSwitchDev {
 		return false, fmt.Errorf("virtio/vdpa requires the device to be configured in switchdev mode")
 	}
+
+	// Externally created: we don't support  ExternallyCreated + EswitchMode
+	//TODO: if needed we will need to add this in the future as today EswitchMode is for HWOFFLOAD
+	if cr.Spec.ExternallyCreated && cr.Spec.EswitchMode == sriovnetworkv1.ESwithModeSwitchDev {
```

Comment:
why do you want to block it ?

if its externally created and device is already in switchdev mode then its OK no ?

Reply:
we should block right now because all the systemd and bash script we do for this will be hard to manage.
and right know I am not aware of any request for this.